packages/@tailwindcss-vite/src/index.ts

```diff
@@ -97,23 +97,30 @@ export default function tailwindcss(opts: PluginOptions = {}): Plugin[] {
       name: '@tailwindcss/vite:generate:serve',
       apply: 'serve',
       enforce: 'pre',
+      transform: {
+        filter: {
+          id: {
+            exclude: [/\/\.vite\//, SPECIAL_QUERY_RE, COMMON_JS_PROXY_RE],
+            include: [/css(?:\?.*)?$/, /&lang\.css/, INLINE_STYLE_ID_RE],
```

Comment:
I think it should be this instead?

```suggestion
            include: [/\.css(?:\?.*)?$/, /&lang\.css/, INLINE_STYLE_ID_RE],
```

We use `path.extname(…)` which grabs the extension after the last `.` in the ID.

Reply:
Good catch ☺️